verification/verify_selection_button.py

```
@@ -0,0 +1,55 @@
+
+        # Verify it is checked
+        expect(cramps_button).to_be_checked()
+
+if __name__ == "__main__":
+    with sync_playwright() as p:
+        browser = p.chromium.launch(headless=True)
+        context = browser.new_context(viewport={"width": 375, "height": 812}) # Mobile viewport
+        page = context.new_page()
+        try:
+            test_selection_button_press_animation(page)
+        except Exception as e:
```

Ruff (0.15.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)